catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/writer/CswRecordCollectionMessageBodyWriter.java

```diff
@@ -62,6 +62,10 @@
     private static final String OCTET_STREAM_OUTPUT_SCHEMA =
             "http://www.iana.org/assignments/media-types/application/octet-stream";
 
+    private static final String HEADER_ACCEPT_RANGES = "Accept-Ranges";
```

Comment:
I don't think we have an existing place that contain these constants, but I can move them into CswConstants.